packages/react-router-dev/vite/cloudflare-dev-proxy.ts

```diff
@@ -4,6 +4,7 @@ import { type Plugin } from "vite";
 import { type GetPlatformProxyOptions, type PlatformProxy } from "wrangler";
 
 import { fromNodeRequest, toNodeRequest } from "./node-adapter";
+import { preloadViteEsm, importViteEsmSync } from "./import-vite-esm-sync";
```

Comment:
This has changed in the `dev` branch.

```suggestion
import { preloadVite, getVite } from "./vite";
```

Reply:
Updated.